plutus-ledger/src/Ledger/Blockchain.hs

```diff
@@ -51,7 +53,15 @@ import           Plutus.V1.Ledger.Value   (Value)
 -- | Block identifier (usually a hash)
 newtype BlockId = BlockId { getBlockId :: BS.ByteString }
     deriving stock (Eq, Ord, Generic)
-    deriving (ToJSON, FromJSON, Show) via LedgerBytes
```

Comment:
This is a bit annoying... I guess it no longer works since we're not exporting the constructors.